vagrant/k8s/prometheus-adapter.yaml

```diff
@@ -0,0 +1,15 @@
+rules:
+  default: false
+  custom:
+  - seriesQuery: 'kube_pod_info{namespace!="",pod!=""}'
+    name:
+      matches: "kube_pod_info"
+      as: "pod_info"
+    resources:
+      overrides:
+        namespace: {resource: "namespace"}
+        pod: {resource: "pod"}
```

Comment:
renames query `namespace` with kubernetes namespace and query `pod` with kubernetes `pod`